python/ecs/ecs-service-with-advanced-alb-config/app.py

```diff
@@ -7,7 +7,7 @@
 )
 
 app = App()
-stack = Stack(app, "aws-ec2-integ-ecs")
```

Comment:
aws-ec2-integ-ecs stack name is concatenated with the construct id for the capacity provider name... the capacity provider name however can't start with 'aws'